superx/routes/home.py

```diff
@@ -7,13 +7,18 @@
 from models import Branch, BranchPrice, Product
 from app import db, supermarket_info_dictionary as sd
 
+#  The amount of items to show the customer each search
+# (more items = bigger delay from the input to the presentation)
+NUMBER_OF_ITEMS_TO_SHOW = 20
+
 
 def home():
     """
     returns landing page of application
     """
     city_list = db.session.query(Branch.city).order_by(Branch.city).distinct().all()
-    return render_template('home.html', city_list=city_list)
+    return render_template('home.html', city_list=city_list,
+                           NUMBER_OF_ITEMS_TO_SHOW=NUMBER_OF_ITEMS_TO_SHOW)
```

Comment:
According to python styling:
the parameter_name should be lowercase, while the constant should remain uppercase.